client/src/lib/hooks/useConnection.ts

```diff
@@ -301,9 +301,6 @@ export function useConnection({
         // Add custom header name as a special request header to let the server know which header to pass through 
         if (authHeaderName.toLowerCase() !== "authorization") {
           headers[authHeaderName] = token;
-          if (headerName) {
-            headers["x-custom-auth-header"] = headerName;
-          }
         } else {
```

Comment:
Here the intention was to strip away the conditional (lines 304 and 306), not the content (line 305).

i.e. replace the above deleted block with...
```typescript
headers["x-custom-auth-header"] = authHeaderName;
```

Reply:
Whoops, I totally missed that.